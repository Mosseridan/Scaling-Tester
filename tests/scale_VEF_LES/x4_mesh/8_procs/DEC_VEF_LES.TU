Statistiques de post resolution

Temps total                       8.62386

