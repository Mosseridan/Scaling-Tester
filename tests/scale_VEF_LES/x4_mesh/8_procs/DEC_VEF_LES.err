Petsc initialization succeeded.
Parallel engine initialized : Comm_Group_Noparallel with 1 processors
Executable: /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
Data file name = DEC_VEF_LES
feenableexcept enabled.
Data file : DEC_VEF_LES
Localisation etude: /home/idanmos/projects/Scaling-Tester/tests/scale_VEF_LES/x4_mesh/8_procs
Nom du cas DEC_VEF_LES
 code : /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
 version : 1.7.6 
Debut de l'execution 
MAIN: Reading and executing data file
Dimension::interpreter : dimension = 3
It treats a calculation with a precision 1e-05
It deals with a calculation using precision format 5
Execution of the interpreter Lire_Tgrid
Reading a mesh which comes from Tgrid
Association of the domain on which the interpreter applies Lire_Tgrid
Reading of the file Tube_coude_463259.msh ...
The total number of elements to read is 463259
3D elements of type Tetrahedron
Reading a comment:
(0

Reading of the dimension of the case:
Dimension 3.

Reading a comment:
(0

The total number of nodes to read is 89710

89710 nodes are read in the area 8


The type of area 9 is 1 (1:fluid, 17:solid)

The total number of faces to read is 950399

Reading a comment:
(0

902637 internal faces are read in the area 10

Reading a comment:
(0

19438 faces are read from the boundary number 11

Reading a comment:
(0

27656 faces are read from the boundary number 12

Reading a comment:
(0

334 faces are read from the boundary number 13

Reading a comment:
(0

334 faces are read from the boundary number 14

Reading a comment:
(0

Reading of a name:
The area 9 is called fluid

Reading of a name:
The area 10 is called interieur

Reading of a name:
The area 11 is called symmetry

Reading of a name:
The area 12 is called wall

Reading of a name:
The area 13 is called outlet

Reading of a name:
The area 14 is called inlet

Elimination of 0 unnecessary nodes in the domain dom
Execution of the interpreter Dilate
Association of the domain on which the interpreter applies Dilate
Expands the domain dom of a factor 0.001
Execution of the interpreter Transformer
Association of the domain on which the interpreter applies Transformer
Reading and interpretation of the function x
Interpretation of the function x Ok
Reading and interpretation of the function y
Interpretation of the function y Ok
Reading and interpretation of the function -z
Interpretation of the function -z Ok
Execution of the interpreter ExtrudeBord
Association of the domain on which the interpreter applies ExtrudeBord
number of 3D nodes :89710
number of 3D faces of inlet :334
number of nodes per faces in 3D :3
boundary maintained symmetry
boundary maintained wall
boundary to eliminate outlet
Execution of the interpreter Raffiner_anisotrope
Refinement of each element into 4 new tetrahedra... 
Association of the domain on which the interpreter applies Raffiner_anisotrope
We have split the tetrahedra ...
END of Raffiner_anisotrope...
  1 NbElem=74816  NbNod=23073
Refinement... OK
Execution of the interpreter Raffiner_anisotrope
Refinement of each element into 4 new tetrahedra... 
Association of the domain on which the interpreter applies Raffiner_anisotrope
We have split the tetrahedra ...
END of Raffiner_anisotrope...
  1 NbElem=1853036  NbNod=552969
Refinement... OK
Decouper : Splitting of a domain
 Domain name to split : box
==============================================
The extreme coordinates of the domain box are:
x is between -0.5334 and -0.4572
y is between -0.1624 and -0.1524
z is between 0 and 0.0381
==============================================
==============================================
The volume cells of the domain box are:
sum(volume cells)= 2.27368e-05
mean(volume cells)= 3.03903e-10
min(volume cells)= 1.36766e-10
max(volume cells)= 1.20348e-09
==============================================
domaine = box
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 925
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 Exchange of parts 0 and 3
The partitioner has generated 8 parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 8 parts:
 Construction of part number 0
  Number of nodes    : 2945
  Number of elements : 9352
  Number of joints   : 2
  Joint 0 PeVoisin 1 NbSommets 46 NbFaces 69 NbElemDist 3216
  Joint 1 PeVoisin 2 NbSommets 59 NbFaces 92 NbElemDist 4521
               Total:    NbSommets 105 NbFaces 161 NbElemDist 7737
Writing part 0 into the binary file box_0000.Zones
Writing of 2945 nodes.
Writing of 9352 elements.
 Construction of part number 1
 Adding of a new joint : 5
  Number of nodes    : 2955
  Number of elements : 9353
  Number of joints   : 4
  Joint 0 PeVoisin 0 NbSommets 46 NbFaces 69 NbElemDist 3616
  Joint 1 PeVoisin 2 NbSommets 39 NbFaces 59 NbElemDist 3346
  Joint 2 PeVoisin 3 NbSommets 66 NbFaces 107 NbElemDist 4412
  Joint 3 PeVoisin 5 NbSommets 0 NbFaces 0 NbElemDist 166
               Total:    NbSommets 151 NbFaces 235 NbElemDist 11540
Writing part 1 into the binary file box_0001.Zones
Writing of 2955 nodes.
Writing of 9353 elements.
 Construction of part number 2
  Number of nodes    : 2956
  Number of elements : 9351
  Number of joints   : 5
  Joint 0 PeVoisin 0 NbSommets 59 NbFaces 92 NbElemDist 3604
  Joint 1 PeVoisin 1 NbSommets 39 NbFaces 59 NbElemDist 3307
  Joint 2 PeVoisin 3 NbSommets 14 NbFaces 16 NbElemDist 1618
  Joint 3 PeVoisin 5 NbSommets 27 NbFaces 37 NbElemDist 2631
  Joint 4 PeVoisin 7 NbSommets 33 NbFaces 49 NbElemDist 2986
               Total:    NbSommets 172 NbFaces 253 NbElemDist 14146
Writing part 2 into the binary file box_0002.Zones
Writing of 2956 nodes.
Writing of 9351 elements.
 Construction of part number 3
 Adding of a new joint : 7
  Number of nodes    : 2947
  Number of elements : 9352
  Number of joints   : 5
  Joint 0 PeVoisin 1 NbSommets 66 NbFaces 107 NbElemDist 4830
  Joint 1 PeVoisin 2 NbSommets 14 NbFaces 16 NbElemDist 1622
  Joint 2 PeVoisin 4 NbSommets 43 NbFaces 66 NbElemDist 3780
  Joint 3 PeVoisin 5 NbSommets 38 NbFaces 56 NbElemDist 3546
  Joint 4 PeVoisin 7 NbSommets 0 NbFaces 0 NbElemDist 172
               Total:    NbSommets 161 NbFaces 245 NbElemDist 13950
Writing part 3 into the binary file box_0003.Zones
Writing of 2947 nodes.
Writing of 9352 elements.
 Construction of part number 4
  Number of nodes    : 2956
  Number of elements : 9352
  Number of joints   : 4
  Joint 0 PeVoisin 3 NbSommets 43 NbFaces 66 NbElemDist 3156
  Joint 1 PeVoisin 5 NbSommets 72 NbFaces 116 NbElemDist 5596
  Joint 2 PeVoisin 6 NbSommets 23 NbFaces 30 NbElemDist 1883
  Joint 3 PeVoisin 7 NbSommets 10 NbFaces 11 NbElemDist 1196
               Total:    NbSommets 148 NbFaces 223 NbElemDist 11831
Writing part 4 into the binary file box_0004.Zones
Writing of 2956 nodes.
Writing of 9352 elements.
 Construction of part number 5
 Adding of a new joint : 1
  Number of nodes    : 2960
  Number of elements : 9352
  Number of joints   : 6
  Joint 0 PeVoisin 1 NbSommets 0 NbFaces 0 NbElemDist 230
  Joint 1 PeVoisin 2 NbSommets 27 NbFaces 37 NbElemDist 1690
  Joint 2 PeVoisin 3 NbSommets 38 NbFaces 56 NbElemDist 3002
  Joint 3 PeVoisin 4 NbSommets 72 NbFaces 116 NbElemDist 4928
  Joint 4 PeVoisin 6 NbSommets 30 NbFaces 42 NbElemDist 2197
  Joint 5 PeVoisin 7 NbSommets 57 NbFaces 85 NbElemDist 5160
               Total:    NbSommets 224 NbFaces 336 NbElemDist 17207
Writing part 5 into the binary file box_0005.Zones
Writing of 2960 nodes.
Writing of 9352 elements.
 Construction of part number 6
  Number of nodes    : 2946
  Number of elements : 9352
  Number of joints   : 3
  Joint 0 PeVoisin 4 NbSommets 23 NbFaces 30 NbElemDist 1889
  Joint 1 PeVoisin 5 NbSommets 30 NbFaces 42 NbElemDist 2658
  Joint 2 PeVoisin 7 NbSommets 61 NbFaces 90 NbElemDist 5993
               Total:    NbSommets 114 NbFaces 162 NbElemDist 10540
Writing part 6 into the binary file box_0006.Zones
Writing of 2946 nodes.
Writing of 9352 elements.
 Construction of part number 7
 Adding of a new joint : 3
  Number of nodes    : 2974
  Number of elements : 9352
  Number of joints   : 5
  Joint 0 PeVoisin 2 NbSommets 33 NbFaces 49 NbElemDist 2608
  Joint 1 PeVoisin 3 NbSommets 0 NbFaces 0 NbElemDist 78
  Joint 2 PeVoisin 4 NbSommets 10 NbFaces 11 NbElemDist 469
  Joint 3 PeVoisin 5 NbSommets 57 NbFaces 85 NbElemDist 4795
  Joint 4 PeVoisin 6 NbSommets 61 NbFaces 90 NbElemDist 4209
               Total:    NbSommets 161 NbFaces 235 NbElemDist 12159
Writing part 7 into the binary file box_0007.Zones
Writing of 2974 nodes.
Writing of 9352 elements.
End of the interpreter Decouper
MAIN: End of data file
Arret des processes.
