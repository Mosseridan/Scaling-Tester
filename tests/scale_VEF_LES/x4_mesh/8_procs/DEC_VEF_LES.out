-------------------------------------------------------------------
 
                          TRUST
                      version : 1.7.6 
                          CEA - DEN
 

 0 file(s) modified 


 405 file(s) created 

 ------------------------------------------------------------------
 
  Vous traitez le cas DEC_VEF_LES
 
Reading data file 

13 MBytes of RAM taken by the calculation.
26 MBytes of RAM taken by the calculation.
26 MBytes of RAM taken by the calculation.
27 MBytes of RAM taken by the calculation.
28 MBytes of RAM taken by the calculation.
28 MBytes of RAM taken by the calculation.
 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 8
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[1.25e-01]   1=[1.25e-01]   2=[1.25e-01]   3=[1.25e-01]   4=[1.25e-01]
        5=[1.25e-01]   6=[1.25e-01]   7=[1.25e-01]
   Allowed maximum load imbalance: 1.001 

     74816     297924     297924 [20] [     5611:   74816 ]
     37889     218168     224070 [20] [     5611:   74816 ]
     20263     119380     185958 [20] [     5611:   74816 ]
     10881      77312     141294 [20] [     5611:   74816 ]
      5827      45340     113080 [20] [     5611:   74816 ]
      3140      24418      89716 [20] [     5611:   74816 ]
      1685      12996      67490 [20] [     5611:   74816 ]
       906       7564      50388 [20] [     5611:   74816 ]
       480       4626      38922 [20] [     5611:   74816 ]
       253       2648      31562 [20] [     5611:   74816 ]
       132       1426      25812 [20] [     5611:   74816 ]
        68        698      21290 [20] [     5611:   74816 ]
        35        308      16902 [20] [     5611:   74816 ]
        19        148      13894 [20] [     5611:   74816 ]
     37408     148768     148768 [20] [     2805:   37408 ]
     19007     108514     111966 [20] [     2805:   37408 ]
     10176      59274      92646 [20] [     2805:   37408 ]
      5483      38394      70254 [20] [     2805:   37408 ]
      2926      22250      55900 [20] [     2805:   37408 ]
      1570      11854      44068 [20] [     2805:   37408 ]
       843       6340      32656 [20] [     2805:   37408 ]
       451       3666      24084 [20] [     2805:   37408 ]
       235       2182      18726 [20] [     2805:   37408 ]
       122       1226      14796 [20] [     2805:   37408 ]
        63        636      11986 [20] [     2805:   37408 ]
        33        310       9790 [20] [     2805:   37408 ]
        17        134       7740 [20] [     2805:   37408 ]
     18705      74256      74256 [20] [     1402:   18705 ]
      9535      53748      55916 [20] [     1402:   18705 ]
      5115      29274      46006 [20] [     1402:   18705 ]
      2747      18968      34594 [20] [     1402:   18705 ]
      1469      10926      27544 [20] [     1402:   18705 ]
       795       5830      21658 [20] [     1402:   18705 ]
       430       3124      16178 [20] [     1402:   18705 ]
       231       1812      11926 [20] [     1402:   18705 ]
       123       1076       9322 [20] [     1402:   18705 ]
        64        578       7442 [20] [     1402:   18705 ]
        35        312       6028 [20] [     1402:   18705 ]
        18        140       4736 [20] [     1402:   18705 ]
     18703      74224      74224 [20] [     1402:   18703 ]
      9531      53672      55880 [20] [     1402:   18703 ]
      5091      29156      45904 [20] [     1402:   18703 ]
      2739      18956      34412 [20] [     1402:   18703 ]
      1464      10990      27412 [20] [     1402:   18703 ]
       793       5776      21496 [20] [     1402:   18703 ]
       423       3052      15916 [20] [     1402:   18703 ]
       227       1780      11644 [20] [     1402:   18703 ]
       123       1062       8876 [20] [     1402:   18703 ]
        64        554       6912 [20] [     1402:   18703 ]
        35        282       5614 [20] [     1402:   18703 ]
        18        124       4516 [20] [     1402:   18703 ]
     37408     148740     148740 [20] [     2805:   37408 ]
     19001     108356     111926 [20] [     2805:   37408 ]
     10181      59214      92594 [20] [     2805:   37408 ]
      5466      38244      70070 [20] [     2805:   37408 ]
      2922      22228      55752 [20] [     2805:   37408 ]
      1573      11836      43880 [20] [     2805:   37408 ]
       837       6214      32796 [20] [     2805:   37408 ]
       443       3588      24516 [20] [     2805:   37408 ]
       236       2158      18790 [20] [     2805:   37408 ]
       124       1206      15262 [20] [     2805:   37408 ]
        65        650      12236 [20] [     2805:   37408 ]
        33        290      10020 [20] [     2805:   37408 ]
        17        124       7852 [20] [     2805:   37408 ]
     18704      74230      74230 [20] [     1402:   18704 ]
      9540      53754      55902 [20] [     1402:   18704 ]
      5117      29196      46036 [20] [     1402:   18704 ]
      2761      18900      34708 [20] [     1402:   18704 ]
      1474      10910      27576 [20] [     1402:   18704 ]
       787       5742      21582 [20] [     1402:   18704 ]
       422       3000      16038 [20] [     1402:   18704 ]
       229       1742      11840 [20] [     1402:   18704 ]
       125       1060       9074 [20] [     1402:   18704 ]
        66        602       7198 [20] [     1402:   18704 ]
        34        316       5836 [20] [     1402:   18704 ]
        18        140       4648 [20] [     1402:   18704 ]
     18704      74174      74174 [20] [     1402:   18704 ]
      9562      53556      55890 [20] [     1402:   18704 ]
      5103      29002      45834 [20] [     1402:   18704 ]
      2741      18720      34426 [20] [     1402:   18704 ]
      1461      10788      27348 [20] [     1402:   18704 ]
       775       5676      21304 [20] [     1402:   18704 ]
       419       3034      15928 [20] [     1402:   18704 ]
       227       1686      11544 [20] [     1402:   18704 ]
       122       1002       8826 [20] [     1402:   18704 ]
        64        532       7056 [20] [     1402:   18704 ]
        33        264       5540 [20] [     1402:   18704 ]
        18        128       4292 [20] [     1402:   18704 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.206
     Coarsening: 		   0.143
            Matching: 			   0.061
            Contract: 			   0.070
     Initial Partition: 	   0.001
     Uncoarsening: 		   0.046
          Refinement: 			   0.025
          Projection: 			   0.020
     Splitting: 		   0.013
********************************************************************

 gk_mcore statistics
           coresize:       898344         nmops:         2048  cmop:      0
        num_callocs:          865   num_hallocs:            0
       size_callocs:     12293088  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:       897808   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

67 MBytes of RAM taken by the calculation.

--------------------------------------------
clock: Total execution: 8.62386 s
